src/routes.tsx

```diff
@@ -12,12 +12,8 @@ const loadModule = ( cb ) => ( componentModule ) => {
 };
```

Comment:
hmm I don't see a point here why loadModule is a closure, when it can be loaded via one function call, see docs